qemu/tests/cfg/cluster_size_check.cfg

```diff
@@ -16,6 +16,11 @@
             status_error = "no"
             # Cluster size must be a power of two between 512 and 2048k.
             cluster_size_set = "512 1K 2K 4K 8K 16K 32K default 128K 256K 512K 1M 2M"
+            variants:
+                - preallocation_default:
```

Comment:
There is no need to specify `preallocation` in the default mode, it should be `None` instead of `off`.
Also, better to add `@` to the variant tag.